crates/biome_cli/tests/snapshots/main_cases_graphql/lint_single_rule.snap

```diff
@@ -10,25 +11,14 @@ query {
 		id
 	}
 }
-```
-
-# Termination Message
-
-```block
-lint ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
-
-  × Some errors were emitted while running checks.
-  
-
-
 ```
 
 # Emitted Messages
 
 ```block
 file.graphql:2:10 lint/nursery/useDeprecatedReason ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 
-  × The directive `@deprecated` should have a `reason` argument.
+  i The directive `@deprecated` should have a `reason` argument.
```

Comment:
This is an expected change because now all rules start with the `Severity::Information` if not severity is set